themes/binaryanomaly/README.md

```diff
@@ -0,0 +1,17 @@
+# Binaryanomaly
```

Comment:
```suggestion
# Binary Anomaly
```

You capitalized the theme name in the heading. In this case, I think it reads better to separate the words. What do you think?